src/docs/development/add-to-app/ios/project-setup.md

```diff
@@ -174,10 +174,7 @@ into your targets's build settings > General > Frameworks, Libraries, and Embedd
   <div class="row">
```

Comment:
you can also just replace this whole thing with a {% include app-figure.md... if you want